recipes/pcl/all/conanfile.py

```diff
@@ -0,0 +1,133 @@
+import os
+from os.path import join
+from glob import glob
+from itertools import chain
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class PclConanRecipe(ConanFile):
+    name = "pcl"
+    description = "Point Cloud Library"
+    license = "BSD-3-Clause"
+    homepage = "https://pointclouds.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_cuda": [True, False],
+        "with_tools": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+        "with_cuda": False,
+        "with_tools": False
+    }
+    requires = ("boost/1.70.0",
+                "eigen/3.3.7",
+                "flann/1.9.1",
+                "libpng/1.6.37",
+                "opengl/system",
+                "qhull/7.3.2")
+    generators = ["cmake", "cmake_find_package"]
+    exports = ["CMakeLists.txt"]
+
+    @property
+    def _source_subfolder(self):
+        return "pcl-pcl-{}".format(self.version)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version])
        os.rename("pcl-pcl-{}".format(self.version), self._source_subfolder)
```

To keep the same equality with other recipes, use source_subfolder by default.